src/index.js

```diff
@@ -322,6 +322,17 @@ class PublicationClient extends EventEmitter {
     delete this._subscriptions[subKey];
   }
 
+  /**
+   * Remove a subscription from the current session.
+   *
+   * @param {String} name The publication to subscribe to.
+   * @param {*[]} params (optional) Params to pass to the publication.
+   */
+  unsubscribe() {
```

Comment:
How's this different from how we stop subscriptions today? https://hound.mixmax.com/?q=subscription.*%5C.stop%5C(%5C)&i=fosho&files=&excludeFiles=&repos=mixmaxhq/app

Reply:
good thinking. I was looking at the index exposed methods. But will try this one out :D it should work great

Reply:
@bradvogel  because when unmounting the page I want to stop subscriptions by key,  not to keep the references of all the subscriptions I started.

Perhaps it is better this method to contain:

```
unsubscribe() {
    const subKey = JSON.stringify(_.toArray(arguments));
    this._subscriptions[subKey].stop();
    this._removeSubscription[subKey];
}
```

Reply:
since the `_removeSubscription` only deletes the key from the `_subscriptions` object

Reply:
@bradvogel what do you think?

Reply:
Changes: 
`getSubscriptionById` method and you can call just `stop`.